validator/accounts/accounts_backup.go

```diff
@@ -48,9 +48,9 @@ func BackupAccountsCli(cliCtx *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "could not initialize wallet")
 	}
-	if w.KeymanagerKind() == keymanager.Remote {
+	if w.KeymanagerKind() == keymanager.Remote || w.KeymanagerKind() == keymanager.Web3Signer {
```

Comment:
the long-term solution is to have an interface called `KeyBackuper` and check if the keymanager implements that interface, instead of all these checks for what it is not. No need to do so in this PR, as we can do that cleanup later with an external contributor that is interested in it. @michaelneuder FYI